easybuild/easyconfigs/m/medaka/medaka-1.8.1-foss-2022a.eb

```
@@ -0,0 +1,69 @@
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Danilo Gonzalez <danilo.gonzalez@hpcnow.com>
+# License::     GPL-v3.0
+# Updated to foss-2020b to use with artic tool
+# J. Sassmannshausen (GSTT/NHS UK)
+# Updated to 1.5.0
+# Jasper Grimm (UoY)
+
+easyblock = 'PythonBundle'
+
+name = 'medaka'
+version = '1.8.1'
+
+homepage = 'https://github.com/nanoporetech/medaka'
+description = "medaka is a tool to create a consensus sequence from nanopore sequencing data."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True}
+
+builddependencies = [('Autotools', '20220317')]
+
+_minimap_ver = '2.24'
+dependencies = [
+    ('Python', '3.10.4'),  # includes cffi
+    # TensorFlow ~=2.8.0 required by medaka 1.8.1, see requirements.txt
+    ('TensorFlow', '2.11.0'),
+    ('Pysam', '0.19.1'),
+    ('SAMtools', '1.16.1'),
+    ('minimap2', _minimap_ver),
+    ('HTSlib', '1.15.1'),  # for tabix, bgzip
+    ('Racon', '1.5.0'),
+    ('edlib', '1.3.9'),
+    ('pyspoa', '0.0.9'),
+    ('python-parasail', '1.3.3'),
+    ('ont-fast5-api', '4.1.1'),
+    ('WhatsHap', '1.7'),
+    ('intervaltree-python', '3.1.0'),
+    ('BCFtools', '1.15.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('mappy', _minimap_ver, {
+        'checksums': ['35a2fb73ef14173283d5abb31e7a318429e0330c3be95851df38dd83d4ff9af9'],
+    }),
+    (name, version, {
+        'checksums': [
+	  '550b696dd4c84c1b8bf674569217004337b8a5d79a71d8cc892fd0461b05d3df', # medaka-1.8.1.tar.gz
+	  '82bbb03df5144263d4a0c00de7921772207f905b6f9c29f34ec2a4c94c3d15ce', # medaka-1.8.1-tensorflow_version.patch,
+	],
+	'patches': ['medaka-1.8.1-tensorflow_version.patch'],
```

It's better to avoid a patch here (so we don't need to drag it along), and just change `requirements.txt` on the fly via `preinstallopts`, something like:

```suggestion
        # remove TensorFlow version requirement which is too strict
        'preinstallopts': "sed -i 's/tensorflow.*/tensorflow/g' requirements.txt && ",
```

(and then also remove the patch)